src/renderer/scss/component/_snack-bar.scss

```diff
@@ -1,30 +1,54 @@
 .snack-bar {
-  padding: $spacing-vertical;
-  position: fixed;
-  top: $spacing-vertical;
+  background-color: var(--color-primary);
+  border-radius: 10px;
+  box-shadow: var(--box-shadow-layer);
+  color: #f0f0f0;
```

Comment:
This should use a text color variable. Probably just `--text-color` or whatever we call it.

Reply:
There's a snackbar specific variable now.